Tools/Schema.NET.Tool/Services/SchemaService.cs

```diff
@@ -43,7 +43,7 @@ public async Task<IEnumerable<GeneratorSchemaObject>> GetObjectsAsync()
             var enumerations = new List<GeneratorSchemaEnumeration>();
             var classes = new List<GeneratorSchemaClass>();
             foreach (var schemaClass in schemaClasses
-                .Where(x => !x.IsPrimitive && !x.IsArchived && !x.IsMeta && !x.IsPending))
```

Comment:
Are you sure we're not going to get pending types added in?

Reply:
I've moved that logic out to the SchemaRepository as it seemed to make sense having it control whether we wanted archived, meta and pending types. This is to also partially set up the path to supporting a separate "pending" schema library.